tests/conftest.py

```diff
@@ -60,6 +61,12 @@ def mock_ktools_read_soa_metadata(mock_read_soa_metadata):
         yield mock_read_soa_metadata
 
 
+@pytest.fixture(autouse=True)
```

Comment:
might be nice to use pytest-antilru eventually, but I think last time I tried to use it it broke a bunch of tests :p